Exiled.API/Features/Player.cs

```diff
@@ -105,6 +105,7 @@ public class Player : IEntity
 
         private ReferenceHub referenceHub;
         private CustomHealthStat healthStat;
+        private CustomHumeShieldStat humeshieldStat;
```

Comment:
Should probably be named `humeShieldStat`, to follow base-game naming scheme of `HumeShield[...]`

Reply:
i have followed what Exiled use for name also it's private so i can't start with H but with h only

Reply:
Sorry, I meant the capital `S` in shield, not the capital H (edited to reflect). The private variable should be named `humeShieldStat`, not `humeshieldStat`.